Backend/app/rider-platform/rider-app/Main/src/Domain/Constants/ChatSuggestions.hs

```diff
@@ -0,0 +1,156 @@
+module Domain.Constants.ChatSuggestions where
+
+import qualified Data.Map.Strict as Map
+import Domain.Types.ChatSuggestions
+import Kernel.Prelude
+
+dataMap :: Map.Map Text Messages
+dataMap =
+  Map.fromList
+    [ ("cis1AP", Messages {en_us = "At pick-up, Where are you ?", ta_in = "பிக்-அப்பில் உள்ளேன், நீங்க?", kn_in = "ಪಿಕಪ್ ನಲ್ಲಿ ಇದ್ದೇನೆ ನೀವು ಎಲ್ಲಿದ್ದೀರಾ?", hi_in = "मैं लोकेशन पे हूं , आप कहा हो ?", ml_in = "ഞാൻ പിക്ക്-അപ്പിൽ ആണ്, താങ്കൾ എവിടെ?", bn_in = "আমি পিক-আপ স্থানে আছি, আপনি কোথায়?"}),
+      ("cis2AP", Messages {en_us = "Reaching shortly", ta_in = "விரைவில் வருகிறேன்", kn_in = "ಶೀಘ್ರದಲ್ಲೇ ತಲುಪುತ್ತೇನೆ", hi_in = "मैं थोड़ी देर में पहुँच रहा हूँ ", ml_in = "ഉടൻ എത്തും", bn_in = "আমি শীঘ্রই পৌঁছাচ্ছি"}),
+      ("cis3AP", Messages {en_us = "Call me, You're unreachable", ta_in = "அணுக முடியவில்லை. கால் செய்யுங்கள்", kn_in = "ಕರೆ ಮಾಡಿ, ನೀವು ತಲುಪಲು ಸಾಧ್ಯವಿಲ್ಲ ", hi_in = "आपका फ़ोन नहीं लग रहा, कॉल कीजिये", ml_in = "എന്നെ വിളിക്കൂ, താങ്കൾ പരിധിക്കു പുറത്താണ്", bn_in = "আমাকে কল করুন, আপনার ফোনে যোগাযোগ করা যাচ্ছে না"}),
+      ("cis1BP", Messages {en_us = "Are you starting ?", ta_in = "நீங்கள் தொடங்குகிறீர்களா?", kn_in = "ನೀವು ಪ್ರಾರಂಭಿಸುತ್ತಿದ್ದೀರಾ?", hi_in = "आप आ रहे है क्या? ", ml_in = "താങ്കൾ പുറപ്പെടുകയാണോ?", bn_in = "আপনি কি আসছেন ?"}),
+      ("cis2BP", Messages {en_us = "Urgent, come soon", ta_in = "அவசரம், சீக்கிரம் வாருங்கள்", kn_in = "ದಯವಿಟ್ಟು ಬೇಗ ಬನ್ನಿ ", hi_in = "अर्जेंट है प्लीज जड़ली आइये", ml_in = "അത്യാവശ്യമാണ്, വേഗം വരൂ", bn_in = "জরুরী আছে, দয়া করে তাড়াতাড়ি আসুন"}),
```

Comment:
Personally I prefer to create strict and clear Haskell types for parsing these strange codes. But if we will not make any backend logic with these codes, then its ok.

Reply:
Also don't like that many codes duplicated